format/view-spec.md

```diff
@@ -144,13 +146,13 @@ This create statement would produce the following `sql` representation metadata:
 | `type`              | `"sql"` |
 | `sql`               | `"SELECT\n    COUNT(1), CAST(event_ts AS DATE)\nFROM events\nGROUP BY 2"` |
 | `dialect`           | `"spark"` |
-| `default-catalog`   | `"prod"` |
-| `default-namespace` | `["default"]` |
-| `field-aliases`     | `["event_count", "event_date"]` |
-| `field-comments`    | `["Count of events", null]` |
 
 If a create statement does not include column names or comments before `AS`, the fields should be omitted.
 
+The `event_count` (with the `Count of events` comment) and `event_date` field aliases must be part of the view version's `schema`.
+
+The `default-catalog` and `default-namespace` of the view version should use the same value across multiple SQL representations.
```

Comment:
I think "must be" rather than "should be", although this can probably be removed entirely because those are no longer part of the representation. It just IS the same across all representations.

Reply:
indeed this can be removed now